src/EnergyPlus/PlantLoopHeatPumpEIR.cc

```diff
@@ -2365,14 +2365,14 @@ void EIRFuelFiredHeatPump::oneTimeInit(EnergyPlusData &state)
                             OutputProcessor::SOVTimeStepType::System,
                             OutputProcessor::SOVStoreType::Average,
                             this->name);
-        // SetupOutputVariable(state,
+        // Setup Output Variable(state,
         //                    "Fuel-fired Absorption Heat Pump Source Side Inlet Temperature",
         //                    OutputProcessor::Unit::C,
         //                    this->sourceSideInletTemp,
         //                    OutputProcessor::SOVTimeStepType::System,
         //                    OutputProcessor::SOVStoreType::Average,
         //                    this->name);
-        // SetupOutputVariable(state,
+        // Setup Output Variable(state,
```

Comment:
The script that scrapes the code building out the output variable file was complaining about the commented SetupOutputVariable calls.  It potentially botches the rudimentary parser.  So just adding spaces so these aren't detected, but they can be quickly fixed back up later if these are enabled.